pkg/sampledriver/provisioner.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 The Kubernetes Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// You may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sampledriver
+
+import (
+	"context"
+
+	_ "k8s.io/klog/v2"
+	cosi "sigs.k8s.io/container-object-storage-interface-spec"
+)
+
+type ProvisionerServer struct {
+	provisioner string
+}
+
+func (s *ProvisionerServer) ProvisionerCreateBucket(ctx context.Context,
+	req *cosi.ProvisionerCreateBucketRequest) (*cosi.ProvisionerCreateBucketResponse, error) {
+
```

Comment:
Is the intention this to be empty? Since the current sample minio provisioner has the functionality of creating buckets, keys etc, maybe as part of this PR we can keep the same functionality
I do like the idea though of having a "bare" sample provisioner so that providers can use as base

Reply:
Yeah, there are other changes coming into this part. Its a WIP still